packages/queries/src/query.ts

```diff
@@ -161,10 +192,7 @@ export class QueryBuilder<
     }
 
     /**
-     * Builds a query that returns exactly one row.
-     * Will throw an error if the query returns 0.
-     *
-     * Also sets the limit to 1.
+     * Alias for {@link firstOrThrow()}
```

Comment:
I would think we'd want to keep this consistent with the other two, so either one of `Alias for {@link firstOrThrow()}` or `@alias {@link firstOrThrow}`